src/logic/community/types.ts

```diff
@@ -63,3 +81,11 @@ export type GetCommunitiesWithTotal<T> = {
   communities: T[]
   total: number
 }
+
+export type GetCommunityMembersResult = {
+  results: CommunityMember[]
+  total: number
+  page: number
+  pages: number
+  limit: number
+}
```

Comment:
This is already typed in the HTTPResponse. You could use `HTTPResponse<PaginatedResponse<CommunityMember>>`